lnwallet/channel.go

```diff
@@ -2709,8 +2753,26 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 			continue
 		}
 
-		processAddEntry(entry, ourBalance, theirBalance, nextHeight,
-			whoseCommitChain, true, mutateState)
+		// Skip the entries that have already had their add commit
+		// height set for this commit chain.
+		addHeights := &entry.addCommitHeights
+		addHeight := addHeights.GetForParty(whoseCommitChain)
+		if addHeight == 0 {
```

Comment:
Looks like the `Add/Remove/FeeUpdate` have the common logic,
- we get the value based on `whoseCommitChain`
- we decide to process it or not based on its value == 0 or not.

Can be simplified?